xc7/archs/artix7/tiles/slicem/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@ project_xray_tile(
   TILE SLICEM
   SITE_AS_TILE
   SITE_TYPES SLICEM/SLICEM
+  EQUIVALENT_SITES SLICEM
```

Comment:
This needs a TODO, yes?  Eventually will be "EQUIVALENT_SITES SLICEM SLICEL"?

Reply:
Yeah, I'll add it as well. Right now only one `equivalent_site` per tile is allowed.

Reply:
Done.